src/dftd4/model.f90

```diff
@@ -14,7 +14,13 @@
 ! You should have received a copy of the Lesser GNU General Public License
 ! along with dftd4.  If not, see <https://www.gnu.org/licenses/>.
 
-!> Definition of the D4 dispersion model for the evaluation of C6 coefficients.
+!> @dir dftd4/model
```

Comment:
This project uses Ford to process the documentation, why do we add doxygen markets here?